Makefile

```diff
@@ -1,5 +1,6 @@
 GO111MODULE := on
-export GO111MODULE
+GIN_MODE := release
```

Comment:
Do we need it on build time or at runtime? If on build time, it's OK, but if at runtime, I'd suggest to set the var in the example itself instead of the makefile.

Reply:
WIthout it, there's a couple of extra lines of debug output that gin puts out when it instantiates the server.
`os.Setenv("GIN_MODE", "release")` doesn't seem to resolve. I removed it from the `Makefile` though.

Reply:
I guess you need to call setenv before calling gin.Default(). Or maybe you can call gin.SetMode(gin.ReleaseMode) (if it works, it'd be preferred).

Reply:
Got this working by using an `init` function and keeping the gin instance inside the handler.